generate_refresh_token.py

```
@@ -0,0 +1,60 @@
+#   Request details: redirect_uri=http://localhost:8080/ flowName=GeneralOAuthFlow
+#   """
+
+
+# Path to your client secrets JSON file
+CLIENT_SECRETS_FILE = "client_secrets.json"
+
+# Required scopes for YouTube upload
+SCOPES = ["https://www.googleapis.com/auth/youtube.upload"]
+
+
+def generate_refresh_token():
```

Ruff (0.8.2)

Missing return type annotation for public function `generate_refresh_token`

Add return type annotation: `None`

(ANN201)